routes/index.js

```diff
@@ -301,10 +363,20 @@ router.get('/admin_current_jobs', function(req, res, next) {
     })
 });
 
-/* GET admin_postajob page */
-/* GET about page */
-router.get('/admin_postajob', function(req, res, next) {
-  res.render('admin_postajob', buildDefaultMessage(req, "admin_postajob"));
+router.get('/edit_story', function(req, res, next) {
```

Comment:
Should ensure that only admins can access these (auth.ensureUserIsAdmin or something like that)